### PR TITLE
Setting up routes for more data

### PR DESCRIPTION
**Pull request by SamanthaCra**
This added temperature, gas, co2, pressure and humidity routes to the backend. These are all tested as working. Just need the devices_id updated so webhook can send correctly. 
image

### CODE REVIEW

File: fullstack/src/app/api/recieve-data/route.js [hunk 1]
```diff
@@ -41,6 +43,31 @@ export const POST = async (request) => {
             send.push(dustResults)
         }
 
+        if (humidity){
```
Comment (TomMc608) [inline fullstack/src/app/api/recieve-data/route.js:46]: This code is fine but if we want to reduce repetition make it better for scalability and flexibility i suggest somthing like this 

```
const data = {
  dust: { key: 'dust', func: DustData },
  humidity: { key: 'humidity', func: HumidityData },
  temperature: { key: 'temperature', func: TemperatureData },

};

for (const [type, { key, func }] of Object.entries(data)) {
  const value = data.uplink_message.decoded_payload[type];
  if (value !== undefined) {
    const results = await func(device_id, value);
    send.push(results);
  }
}  
```

**Review by TomMc608 — APPROVED** [review]
looks good just one suggestion on a better code for all the if statements but other then that nice work!!

**Review by fuzzyers — APPROVED** [review]
All looks good i dont have much to say about it. Do we have a function made up in the database for getting average temperature

**Comment by SamanthaCra** [comment]
> All looks good i dont have much to say about it. Do we have a function made up in the database for getting average temperature
Yes it is set up, I will document it in the database wiki.

**Pull request merged**